src/services/authenticator.js

```diff
@@ -75,10 +76,18 @@ function Authenticator() {
     });
   };
 
-  this.loginWithEmailOrTokenArgv = async (config) => {
-    const token = await this.login(config);
-    this.saveToken(token);
-    return null;
+  this.tryLogin = async (config) => {
+    await this.logout({ log: false });
+    try {
+      const token = await this.login(config);
+      this.saveToken(token);
+      logger.info('Login successful');
+    } catch (error) {
+      const message = error.message === 'Unauthorized'
+        ? 'Incorrect email or password.'
+        : `${ERROR_UNEXPECTED} ${chalk.red(error)}`;
+      logger.error(message);
```

Comment:
[Thinkin] We could maybe ask again for username / password if the user missed. Just a though, this is ignorable

Reply:
I understand. 
The other hand is when the command ends, we can test it.
(Today we do not know how to test dialog that never end)